compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/ReshapeFusion.cpp

```diff
@@ -254,13 +254,20 @@ static std::optional<SmallVector<Value>> fuseAttentionWithReshapeByExpansion(
     }
   }
 
+  Value maskOperand;
+  if (expandedOpOperands.size() > 4) {
+    maskOperand = expandedOpOperands[4];
+  } else {
+    maskOperand = Value();
+  }
```

Comment:
You don't need the else, the default value is the 'null' state

Reply:
That makes sense